test/CMakeLists.txt

```diff
@@ -107,3 +92,23 @@ target_link_libraries(
 )
 
 add_subdirectory(python)
+
+if (DEFINED NGRAPH_TF_INSTALL_PREFIX)
+    set(CMAKE_INSTALL_PREFIX ${NGRAPH_TF_INSTALL_PREFIX})
+else()
+    set(CMAKE_INSTALL_PREFIX "${CMAKE_CURRENT_BINARY_DIR}/../install/")
+endif()
+
+set(NGTF_INSTALL_BIN_DIR ${CMAKE_INSTALL_PREFIX}/bin)
+
+message(STATUS "NGTF_INSTALL_BIN_DIR: ${NGTF_INSTALL_BIN_DIR}")
```

Comment:
Is `NGTF_INSTALL_BIN_DIR` being used anywhere?

Reply:
No. Will remove this.